server/game/cards/01_SOR/units/FightersForFreedom.ts

```diff
@@ -0,0 +1,31 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { Aspect, CardType } from '../../../core/Constants';
+
+export default class FightersForFreedom extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '5907868016',
+            internalName: 'fighters-for-freedom'
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addTriggeredAbility({
+            title: 'When you play another aggression card: You may deal 1 damage to a base',
```

Comment:
nit: we don't include the "when" or the "you may" parts in the title, since the game will manage the flow for those. So it should just be `Deal 1 damage to a base`

Reply:
Updated